examples/mock/mockoon-bbidentity.json

```diff
@@ -273,6 +273,108 @@
       ],
       "enabled": true,
       "responseMode": null
+    },
+    {
+      "uuid": "91963d01-81ab-4fdd-8b43-9a08652e6173",
+      "documentation": "The authorize endpoint of Open ID Connect (OIDC)",
+      "method": "get",
+      "endpoint": "authorize",
+      "responses": [
+        {
+          "uuid": "7fcecddc-a8e7-41f9-b59b-43933a64efaf",
+          "body": "",
+          "latency": 0,
+          "statusCode": 200,
+          "label": "Successfully loads JS application, and validates the provided query parameters using oauth-details endpoint",
+          "headers": [],
+          "bodyType": "INLINE",
+          "filePath": "",
+          "databucketID": "",
+          "sendFileAsBody": false,
+          "rules": [
+            {
+              "target": "query",
+              "modifier": "scope",
+              "value": "^openid profile$|^openid$|^profile$|^email$|^address$|^phone$|^office_access$",
```

Comment:
Looking at API should be offline_access - not office_access
![image](https://user-images.githubusercontent.com/119664031/213434752-c59544c1-fef4-4fc2-831d-91fe7b0b6e74.png)